template_repo/opg-data-template/integration_tests/v1/conftest.py

```diff
@@ -0,0 +1,111 @@
+import json
+import os
+from os.path import join, dirname
+
+import boto3
+import requests
+from boto3.session import Session
+from jsonschema import validate, exceptions
+from requests_aws4auth import AWS4Auth
+
+template_config = {"name": "template", "url": "http://example-url.com"}
+
+configs_to_test = []
+
+
+def pytest_html_report_title(report):
+    report.title = "template integration tests"
+
+
+def send_a_request(
+    test_config,
+    url=None,
+    method=None,
+    payload=None,
+    extra_headers=None,
+    content_type=None,
+):
+    print(f"Using test_config: {test_config['name']}")
+
+    headers = {
+        "Content-Type": content_type if content_type else "application/json",
+    }
+
+    if extra_headers:
+        for h in extra_headers:
+            headers[h["header_name"]] = h["header_value"]
+
+    if payload:
+        body = json.dumps(payload)
+    else:
+        body = None
+
+    if "CI" in os.environ:
+        role_name = "sirius-ci"
+    else:
+        role_name = "operator"
+
+    boto3.setup_default_session(region_name="eu-west-1",)
```

Comment:
We can make this better. Jack's functions looks cleaner and is more multi purpose (see data-deputy-reporting int tests)